### ISSUE TITLE
aria-label="undefined" added to SVG when no label is set

### ISSUE THREAD

**Issue by samueldmeyer** [czeckd/angular-svg-icon#161]
When an `<svg-icon>` is used as provided in the documentation, `aria-label="undefined"` is added to the svg element. Based on the documentation, this should instead be `aria-hidden="true"`. This is a change between version 17 and 18. For now, the workaround is to set `svgAriaLabel` to an empty string to remove the label.

Reproduction is here:
https://stackblitz.com/edit/er7oo8?file=src%2Fexample%2Fcard-overview-example.html

**Comment by czeckd** [comment]
@samueldmeyer - This should now be fixed in v18.0.2 on npm.

**Comment by samueldmeyer** [comment]
Thank you! It is working in my tests.
